.github/pull_request_template.md

```diff
@@ -0,0 +1,6 @@
+[//]: # "Please remove these comments"
+
+[//]: For general information, please read <"https://github.com/eclipse-ee4j/glassfish/blob/master/CONTRIBUTING.md">.
+
+[//]: If you want to create a PR for gh-pages, please read <"https://github.com/eclipse-ee4j/glassfish/blob/master/docs/README.md">.
```

Comment:
I would rephrase this as...

*DO NOT* create a PR for the gh-pages branch.  Instead, please read [the docs directory README file](https://github.com/eclipse-ee4j/glassfish/blob/master/docs/README.md) and create a PR for the appropriate docs subdirectory in the master branch.

Reply:
done